docker/liberty-testnet-spec.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "LibertyTestnetPoA",
+  "engine": {
+    "authorityRound": {
+      "params": {
+        "stepDuration": "5",
+        "validators": {
+          "list": [
+          ]
+        }
+      }
+    }
+  },
+  "params": {
+    "gasLimitBoundDivisor": "0x400",
+    "maximumExtraDataSize": "0x20",
+    "minGasLimit": "0x1388",
+    "networkID": "0x2323",
+    "chainID": "0x75E",
+    "validateChainIdTransition": 0,
+    "validateReceiptsTransition": 0,
+    "eip98Transition": 0,
+    "eip140Transition": 0,
+    "eip145Transition": 0,
+    "eip150Transition": 0,
+    "eip155Transition": 0,
+    "eip160Transition": 0,
+    "eip161abcTransition": 0,
+    "eip161dTransition": 0,
+    "eip210Transition": 0,
+    "eip211Transition": 0,
+    "eip214Transition": 0,
+    "eip658Transition": 0,
+    "eip1014Transition": 0,
+    "eip1052Transition": 0,
+    "eip1706Transition": 0,
+    "eip1283Transition": 0,
+    "eip1283DisableTransition": 0,
+    "eip1283ReenableTransition": 0,
+    "eip1344Transition": 0,
+    "eip1884Transition": 0,
+    "eip2028Transition": 0,
+    "eip2315Transition": 0
+  },
+  "genesis": {
+    "seal": {
+      "authorityRound": {
+        "step": "0x0",
+        "signature": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+      }
+    },
+    "difficulty": "0x20000",
+    "gasLimit": "0x989680"
+  },
+  "accounts": {
+    "0x0000000000000000000000000000000000000001": {
+      "balance": "1",
+      "builtin": {
+        "name": "ecrecover",
+        "pricing": {
+          "linear": {
+            "base": 3000,
+            "word": 0
+          }
+        }
+      }
+    },
+    "0x0000000000000000000000000000000000000002": {
+      "balance": "1",
+      "builtin": {
+        "name": "sha256",
+        "pricing": {
+          "linear": {
+            "base": 60,
+            "word": 12
+          }
+        }
+      }
+    },
+    "0x0000000000000000000000000000000000000003": {
+      "balance": "1",
+      "builtin": {
+        "name": "ripemd160",
+        "pricing": {
+          "linear": {
+            "base": 600,
+            "word": 120
+          }
+        }
+      }
+    },
+    "0x0000000000000000000000000000000000000004": {
+      "balance": "1",
+      "builtin": {
+        "name": "identity",
+        "pricing": {
+          "linear": {
+            "base": 15,
+            "word": 3
+          }
+        }
+      }
+    },
+    "0x20f2478FecAbD3D9aA7e13e1D961ED53c9161574": { "balance": "10000000000000000000000" },
```

Comment:
I'm curious, do we need this part of the spec here? The other bits are important, but I don't think this is.

Reply:
yeah this is a good point. I supposed we do not need this.